src/test/java/com/agileavengers/icuconnectbackend/service/ICommunityServiceTest.java

```diff
@@ -221,11 +242,11 @@ void getCommunity() {
     void getCommunityEmpty() {
 //        Instructor instructor = Instructor.builder().name("Test Instructor").build();
```

Comment:
Do you need this here?

Reply:
you are right. the commented functionality is not needed anymore.